### PR TITLE
Create vhardik.json

### PR DESCRIPTION
**Pull request by Hardik-Vanani**
Wanna Add my CNAME

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by saxophone-guy** [comment]
Your subdomain cannot have capital letters. Presuming the CNAME given in your json file is your website, it is unreachable (at least for me). Try adding a screenshot to the PR if you prefer not to make it public just yet.

File: domains/Hardik.json
Comment (CuteDog5695) [inline domains/Hardik.json:1]: Domain needs to be all lower case, but someone already took hardik.is-a.dev. Please choose another subdomain name

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> Your subdomain cannot have capital letters. Presuming the CNAME given in your json file is your website, it is unreachable (at least for me). Try adding a screenshot to the PR if you prefer not to make it public just yet.

He gave his repo in the files, so I could visit the page from that (since its username.github.io/reponame if repo isnt the github.io domain)

File: domains/Hardik.json
Comment (Hardik-Vanani) [reply domains/Hardik.json:1]: ok i have changed the file name to "vhardik".

**Comment by CuteDog5695** [comment]
You need to change the json file name so that you actually change the domain you want, not by the name of the PR

File: vhardik
Comment (CuteDog5695) [inline vhardik:1]: File needs to be in domains directory and be json file. While you do that please delete the Hardik.json file

**Review by iostpa — DISMISSED** [review]

**Pull request merged**